dev/com.ibm.ws.security.jwt/resources/com/ibm/ws/security/jwt/internal/resources/JWTMessages.nlsprops

```diff
@@ -361,3 +361,24 @@ ERROR_BUILDING_SIGNED_JWE.useraction=Check the error that is included in the mes
 KEY_MANAGEMENT_KEY_NOT_FOUND=CWWKS6061E: The [{0}] JWT builder cannot create a JSON Web Encryption (JWE) token because a key management key cannot be found. They key management key alias is [{1}], and the configured trust store is [{2}].
 KEY_MANAGEMENT_KEY_NOT_FOUND.explanation=A key management key is a key that is used to encrypt the Content Encryption Key of a JWE token. A key management key must be specified so the JWT builder can create JWE tokens. A key management key can be specified by using the key alias and configured trust store values that are specified in the message or by using the JwtBuilder API.
 KEY_MANAGEMENT_KEY_NOT_FOUND.useraction=Verify that a key with the alias that is specified in the message exists in the trust store that is specified in the message. If the JwtBuilder API is used, ensure that a key management key is provided.
+
+# Do not translate "MicroProfile Config", "mp.jwt.verify.publickey.location"
+# 0=mp.jwt.decrypt.key.location
+DECRYPT_KEY_LOCATION_INLINE_KEY=CWWKS6062E: The value for the [{0}] MicroProfile Config property appears to include a private key string, which is not a supported configuration.
```

Comment:
Is there a reason this message says "appears to include..." rather than just "includes.."? Seems unnecessarily ambiguous, especially from a translators perspective. Maybe:

`The value for the [{0}] MicroProfile Config property includes a private key string.  Private key string values are not supported for this property.`

Reply:
I say "appears to include" because I'm checking a string to see if it contains `"BEGIN "`. Strictly speaking, a server admin could have their private key in, say, a directory `/some/path/BEGIN FOLDER/with/keys/`. In that case, our check would emit this error message and fail out even though the key location string doesn't actually include a private key.

Reply:
OK that makes sense. If I understand correctly, just the _appearance_ of a private string is what draws the error, so a configuration that includes such an appearance will error out regardless. I can't think of a better way to succinctly convey that than what you have here.